### DISCUSSION TITLE
Bug delete revision

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#17174] [Accessibility]
Hi, When I edit a message and try to delete the previous revision of the message, I get what you see in the image.

The message is not seen correctly.

Pc: Macbook pro 13''
Browser: Google chrome (Version 101.0.4951.64)

<img width="141" alt="Schermata 2022-05-23 alle 14 46 33" src="https://user-images.githubusercontent.com/20476002/169822602-947bbd28-73fe-48c6-915c-7bf2ad43c353.png">

**Comment by benjiallen** [comment]
@Angelk90 thanks for the feedback. Can you give me a little bit more information about how to reproduce this issue? What kind of message are you referring to? Which feature does this message belong to? Can you give me a step by step guide?

**Reply by Angelk90** [reply]
https://user-images.githubusercontent.com/20476002/171611519-da4b53f8-7f22-44e7-9913-abd6fe2adeba.mov

**Reply by benjiallen** [reply]
Awesome. Very helpful. Thank you.

**Comment by cehfisher** [comment]
Hi @Angelk90 - thank you for providing this feedback! After reviewing your issue report against the latest UI, this issue appears to be resolved now. Could you please confirm and close this issue? Thank you! (cc @queenofcorgis)
